src/routes/commands/commands_resource.ts

```diff
@@ -10,88 +10,88 @@ interface Command {
 export const commands: Command[] = [
     {
         name: "Ban",
-        tooltip: "Bans specified user",
+        tooltip: "Bans the specified user but does not delete their recent messages",
         quick: "-mod ban @Samoxive#1234 Spam | 3 days",
         description:
-            "Bans the mentioned user, if a text is given after the mention, " +
-            "that text is used as the ban reason. To ban a user temporarily, you can specify " +
-            'the ban duration by using the seperator "|" and typing the duration after.',
+            "Bans the specified user, if additional text follows the identifier, " +
+            "that text is used as the reason. To ban a user temporarily, you can specify " +
+            'a duration by using the separator "|", and then typing the duration in full.',
         usages: [
             "-mod ban @user",
             "-mod ban @user <reason>",
-            "-mod ban @user <reason> | <time>",
-            "-mod ban <username> <reason> | <time>",
-            "-mod ban <user_id> <reason> | <time>"
+            "-mod ban @user [reason] | <time>",
+            "-mod ban <username> [reason] [| <time>]",
+            "-mod ban <user_id> [reason] [| <time>]"
         ],
         examples: [
             "-mod ban @Samoxive#1234",
             "-mod ban @Samoxive#1234 Spamming",
-            "-mod ban @Samoxive#1234 Messaging invite links | 1 day",
-            "-mod ban Samox | 3 days",
+            "-mod ban @Samoxive#1234 Posting invite links | 1 day",
+            "-mod ban Samoxi | 3 days",
             "-mod ban 145456746721312768"
         ]
     },
     {
         name: "Hardban",
-        tooltip: "Hardbans specified user, deleting their messages",
+        tooltip: "Bans the specified user and deletes their recent messages",
         quick: "-mod hardban @Samoxive#1234 Spam",
         description:
-            "Hardbans the mentioned user, banning them and deleting their messages in the last week. " +
-            "If a text is given after the mention, that text is used as the hardban reason.",
+            "Bans the specified user, banning them and deleting every message they sent in the last week. " +
+            "The syntax is equivalent to that of the command `ban`.",
```

Comment:
Hardban's syntax isn't exactly equilavent as there isn't an argument for duration, maybe we could omit this.

Reply:
Will fix